### DISCUSSION TITLE
Error: unbalanced parenthesis

### DISCUSSION THREAD

**Discussion by zeitstein** [sibiraj-s/vscode-scss-formatter#9] [General]
Hi,

Doesn't seem like a formatting issue, so I'm posting here. The following SCSS rule gives me the "Unbalanced parenthesis" error
```
.class {
  width: min(91vw, var(--max-width));
}
```
Any suggestions? Thanks!

**Comment by sibiraj-s** [comment]
Formatting is actually done with [prettier](https://github.com/prettier/prettier). I could reproduce this in published build but not in dev build, could be a regression from the last release which involves switching to webpack. I will investigate further and update back.

**Reply by sibiraj-s** [reply]
Published v2.3.1. Let me know if that works.

**Reply by zeitstein** [reply]
Yup, that fixed it. Thank you for the quick response!

**Reply by sibiraj-s** [reply]
Awesome 👍
